docs/advanced/mixin/4.shadow.md

```diff
@@ -0,0 +1,46 @@
+@Shadow
+======
+
+Shadow is a technology that redirects the access of the fields and methods from the mixin classes to the vanilla classes. It allows you to use those elements like normal ones so that your method code can be simplified. It can also help you to access the private fields or methods in the target class. 
```

Comment:
This also needs a reword. Shadow acts like a placeholder for the actual field or method call. I wouldn't call it necessarily a redirect because of the bytecode merging replaces the calls.